test failing on master expected 'Camel Core, Eclipse Vert.x, RESTEasy JAX-RS' to contain 'Camel Quarkus Core'

```
1) Project generation tests
       should display input values from previously generated project (with extensions):
     AssertionError: expected 'Camel Core, Eclipse Vert.x, RESTEasy JAX-RS' to contain 'Camel Quarkus Core'
```

is the modification from `Camel Quarkus Core` to `Camel Core` expected? If yes, just need to update the test

--- a/src/test/vscodeUiTest/suite/projectGenerationTest.ts
+++ b/src/test/vscodeUiTest/suite/projectGenerationTest.ts
@@ -304,7 +304,7 @@ describe('Project generation tests', function() {
     await wizard.sendKeys(Key.DOWN, Key.UP);
     const quickPickItemText: QuickPickItemInfo = await wizard.getNthQuickPickItemInfo(0);
     expect(quickPickItemText.label).to.have.string('Last used');
-    expect(quickPickItemText.detail).to.have.string('Camel Quarkus Core');
+    expect(quickPickItemText.detail).to.have.string('Camel Core');
     expect(quickPickItemText.detail).to.have.string('Eclipse Vert.x');
 
     await wizard.cancel();

